cloudformation/template.yaml

```diff
@@ -65,6 +65,9 @@ Parameters:
     Type: String
     Description: The email address where user submitted feedback notifications get sent.
     Default: ''
+    MaxLength: 254
+    AllowedPattern: ^$|^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$
```

Comment:
Could you update the regexp to align with [this](https://github.com/awslabs/aws-api-gateway-developer-portal/blob/staging/dev-portal/src/pages/Admin/Accounts/PendingInvites.jsx#L182-L189)?